agora/Agora/Proposal/Time.hs

```diff
@@ -388,74 +388,97 @@ currentProposalTime = phoistAcyclic $
         mkTime = phoistAcyclic $ plam $ pcon .* PCurrentTime
     pure $ pliftA2 # mkTime # lowerBound # upperBound
 
-{- | True if the 'PProposalTime' is in the draft period.
+{- | Represent relation between current time and a given period.
 
-     @since 0.1.0
+     @since 1.0.0
 -}
-isDraftPeriod ::
-  forall (s :: S).
-  Term
-    s
-    ( PProposalTimingConfig
-        :--> PProposalStartingTime
-        :--> PProposalTime
-        :--> PBool
+data PTimingRelation (s :: S)
```

Comment:
Please add a comment that `PBefore` is not present as such a case results in an error.